utils/storage.py

```diff
@@ -28,8 +28,12 @@ def get_status_path(model_dir):
 
 
 def get_status(model_dir):
+    if torch.cuda.is_available():
+        map_location=lambda storage, loc: storage.cuda()
```

Comment:
Did you try `map_location='cuda'`?

Reply:
I followed the documentation

https://pytorch.org/docs/stable/generated/torch.load.html

```
# Load all tensors onto GPU 1
>>> torch.load('tensors.pt', map_location=lambda storage, loc: storage.cuda(1))
```

Reply:
I saw it but I asked you because in the code you wrote `storage.cuda()` whereas in the documentation it is written `storage.cuda(1)`.

Reply:
Well if the destination GPU id is not specified then it defaults to the current device.

https://pytorch.org/docs/stable/storage.html

Reply:
Okay thanks for the details!